helm/adelphi/templates/workflow-adelphi.yaml

```diff
@@ -26,6 +27,7 @@ spec:
           template: cassandra-image-template
       - name: build-cassandra-mgmt-api
         depends: "build-cassandra"
+        when: "{{ default false .Values.target.commit_hash }}"
```

Comment:
We have a single value "commit_hash" to represent both the C* tag/commit/whatever and the similar entity for the management API.  Is it valid that both C* and the mgmt API will use the same tag names... or do we need to allow users to specify distinct "commit hash" values, one for C* and one for the management API?

Reply:
I'm not sure I understand your question here, I think there's some confusion on how the Operator works. Please, check the details I commented below and let me know if this is still a valid concern.